client/state/index.js

```diff
@@ -47,5 +47,8 @@ export function createReduxStore() {
 	) {
 		createStoreWithMiddleware = compose( createStoreWithMiddleware, window.devToolsExtension() );
 	}
-	return createStoreWithMiddleware( createStore )( reducer );
-};
+	if ( initialState === null ) {
+		initialState = {};
+	}
```

Comment:
:white_check_mark: Updated to remove this in 675daac, since I explicitly check for this in localforage.getItem handler